Utilities/DatabasePorter/api.go

```diff
@@ -130,7 +130,7 @@ func GetEntry(hash string) (interfaces.IEBEntry, error) {
 		}
 		return entry, nil
 	}
-	panic("Failed 100 times to get the data " + hash)
+	//panic("Failed 100 times to get the data " + hash)
 	return nil, nil
 }
 
```

Comment:
No, if you keep getting errors, you will keep doing `continue` until you run out of the loop and then you return nil nil